public/location-autocomplete-canonical-list.json

```diff
@@ -64,7 +64,7 @@
   ["Cuba", "country:CU"],
   ["Curaçao", "territory:CW"],
   ["Cyprus", "country:CY"],
-  ["Czechia", "country:CZ"],
+  ["Czech Republic", "country:CZ"],
```

Comment:
I assume you can still use Czechia as a synonym?

Reply:
Yes, the name will show as `Czech Republic`, but the synonym `Czechia` remains.